weblab/static/js/experiment.js

```diff
@@ -0,0 +1,995 @@
+var showdown = require('showdown');
+var notifications = require('./lib/notifications.js');
+var utils = require('./lib/utils.js')
+var expt_common = require('./expt_common.js');
+var plugins = [
+  //require('./visualizers/displayBivesDiff/displayBivesDiff.js'),
+  require('./visualizers/displayContent/displayContent.js'),
+  require('./visualizers/displayPlotD3/displayPlotD3.js'),
```

Comment:
I should have said this before actually - we're deprecating this visualiser, so if anything in it breaks just delete it!